src/anbox/container/lxc_container.cpp

```diff
@@ -76,6 +78,53 @@ void LxcContainer::setup_id_maps() {
                                        max_id - creds_.gid() - 1));
 }
 
+void LxcContainer::setup_network() {
+  if (!fs::exists("/sys/class/net/anbox0")) {
+    WARNING("Anbox bridge interface 'anbox0' doesn't exist. Network functionality will not be available");
+    return;
+  }
+
+  set_config_item("lxc.network.type", "veth");
+  set_config_item("lxc.network.flags", "up");
+  set_config_item("lxc.network.link", "anbox0");
+
+  // Instead of relying on DHCP we will give Android a static IP configuration
+  // for the virtual ethernet interface LXC creates for us. This will be bridged
+  // to the host and will allows us to have reliable network connectivity and
+  // not depend on any other system service.
+  //
+  // See http://androidxref.com/7.1.1_r6/xref/frameworks/base/core/java/android/net/IpConfiguration.java
+  // for more details of the IP configuration format used here.
+
+  android::IpConfigBuilder ip_conf;
+  ip_conf.set_version(android::IpConfigBuilder::Version::Version2);
+  ip_conf.set_assignment(android::IpConfigBuilder::Assignment::Static);
+  ip_conf.set_link_address("192.168.250.2", 24);
+  ip_conf.set_gateway("192.168.250.1");
+  ip_conf.set_dns_servers({"8.8.8.8"});
```

Comment:
Will add proper constants for these. Maybe also make the DNS servers configurable.